packages/pancake-uikit/src/components/MenuItem/styles.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import { MenuItemProps } from "./types";
+
+type StyledMenuItemProps = Pick<MenuItemProps, "isActive">;
+
+const StyledMenuItem = styled.a<StyledMenuItemProps>`
+  display: block;
+  color: ${({ theme }) => theme.menuItem.textColor};
+  font-size: 16px;
+  font-weight: ${({ isActive }) => (isActive ? "600" : "400")};
+  padding: 8px 16px;
+
+  &:hover {
+    background: #eff4f5;
```

Comment:
Will this work on dark and light?

Reply:
Did not handle dark mode atm, working on it.